Tracking/src/ClicEfficiencyCalculator.cc

```diff
@@ -498,10 +500,17 @@ void ClicEfficiencyCalculator::processEvent( LCEvent* evt ) {
           TLorentzVector particleVector2;
           particleVector2.SetPxPyPzE(particle2->getMomentum()[0],particle2->getMomentum()[1],particle2->getMomentum()[2],particle2->getEnergy());
           double DR = particleVector.DeltaR(particleVector2);
+          distancesR.push_back(DR);
           if (DR<0.4) nCloseTrk++;
         }
       }
     }
+    double minDR = distancesR.at(0);
```

Comment:
You could use the min_element algorithm
http://en.cppreference.com/w/cpp/algorithm/min_element